spec/models/start_teacher_training_spec.rb

```diff
@@ -28,4 +28,19 @@
       expect(starter.next_step).to eq("date_of_birth")
     end
   end
+
+  describe "#year_range" do
+    subject { starter.year_range(2) }
+
+    it "returns an array of OpenStructs" do
```

Comment:
Not sure if this test adds value - it's covered in the following test anyway and the fact it's an open struct doesn't really matter?